niworkflows/utils/spaces.py

```diff
@@ -660,10 +660,14 @@ def __call__(self, parser, namespace, values, option_string=None):
             spaces.checkpoint()
         for val in values:
             val = val.rstrip(":")
-            # Should we support some sort of explicit "default" resolution?
-            # https://github.com/nipreps/niworkflows/pull/457#discussion_r375510227
-            # if ":res-" not in val or ":resolution-" not in val:
-            #     val = ":".join((val, "res-default"))
+            if (
+                val not in NONSTANDARD_REFERENCES
+                and not val.split(':')[0].startswith('fs')
+                and ":res-" not in val
+                and ":resolution-" not in val
+            ):
+                # by default, explicitly set volumetric resolution to native
+                val = ":".join((val, "res-native"))
```

Comment:
@effigies res-native is already handled correctly, so this change just appends it to the output spaces by default if no resolution is specified